forc/src/cli/mod.rs

```diff
@@ -28,6 +28,32 @@ mod commands;
 mod plugin;
 pub mod shared;
 
+/// Creates a new project and install their dependencies
+#[cfg(any(feature = "test", test))]
```

Comment:
Isn't this the same as:

```suggestion
#[cfg(test)]
```